discord/ext/commands/pagination.py

```diff
@@ -0,0 +1,58 @@
+import discord, asyncio
+from typing import List, Union
+
+from discord import abc
+from discord.ui.button import button
+
+class Paginate(discord.ui.View):
+    """Creates a paginator for embeds that is navigated with buttons.
+    
+    Parameters
+    ------------
+    send_to: :class:`discord.abc.Messageable`
+    embed: List[:class:`discord.Embed`]
+    """
+
+    def __init__(self, send_to, embeds: List[discord.Embed]):
```

Comment:
`messageable` makes more sense to me than `send_to` - Would also be nice to type hint.

Reply:
I can definitely change it to `messageable`
I actually had a typehint there originally and it would cause my `isinstance` to always pass. It was strange